src/gdpr.erl

```diff
@@ -1,18 +1,18 @@
 -module(gdpr).
 
 -export_type(
-    [binary_table_name/0,
+    [table_name/0,
```

Comment:
I'm actually wondering if we should call it "table" because the returned schema or data doesn't have to necessarily reflect actual, specific table in DB. Maybe a name like "group" would be more accurate (like "data group). Or "type".

Reply:
I'll rename it to `table`

Reply:
I'll rename it in all 3 PRs #2280, #2281, #2282